app/frontend/actions/wallet.ts

```diff
@@ -208,6 +210,19 @@ export default (store: Store) => {
     }
   }
 
+  let lastReloadTime = new Date()
```

Comment:
In theory, lastReloadTime should probably be bumped by `reloadWalletInfo()` and it should be initially set to some value indicating the wallet wasn't reloaded yet ever, like `new Date(0)`  - this way we would ensure that the wallet as such gets reloaded every 30 seconds even if other parts of the app use the non-debounced version of the function, or did you intentionally bump it here?

Reply:
and I'd name the variable `_lastWalletReloadTime` to make it a bit more explicit

Reply:
-> discussed offline